package cn.itcast_02;
/**
 * 我们自己如何处理异常：
 * A: try...catch...finally
 * B: throws 抛出
 * 
 *     try{
 *    		可能出问题的代码;
 *    }catch(异常名 变量){
 *    		问题处理;
 *    }finally{
 *    		释放资源;
 *    }
 *    
 * 
 *    	try{
 *    		....
 *      }catch(异常名 变量){
 *    		....
 *       }catch(异常名 变量){
 *       	.....
 *       }
 *       
 *       try 里出现问题，抛出异常和catch匹配，一旦匹配成功，继续往下执行
 *    
 */
public class ExceptionDemo {
	
}
